tests/functional/test_paramfile.py

```diff
@@ -67,3 +68,24 @@ def test_does_use_fileb_prefix(self):
             provided_param=param,
             expected_param=b'file content'
         )
+
+
+class TestCLIUseEncodingFromEnv(BaseTestCLIFollowParamFile):
+
+    def test_does_use_encoding_utf8(self):
+        path = self.files.create_file('foobar.txt', '經理')
+        param = 'file://%s' % path
+        with patch.dict(os.environ, {'AWS_CLI_FILE_ENCODING': 'utf-8'}):
+            self.assert_param_expansion_is_correct(
+                provided_param=param,
+                expected_param='經理'
+            )
+
+    def test_does_use_encoding_cp1251(self):
+        path = self.files.create_file('foobar.txt', '經理')
+        param = 'file://%s' % path
+        with patch.dict(os.environ, {'AWS_CLI_FILE_ENCODING': 'cp1251'}):
```

Comment:
We should be able to set the env var in the `self.environ` property on the test as we already subclass from this [class](https://github.com/aws/aws-cli/blob/0933b44a9784913784773d21aad9ab5e72884190/awscli/testutils.py#L346) that patches it for us.